Makefile

```diff
@@ -154,6 +154,8 @@ $(BUILD_DIR)/kubernetes-split-yaml: $(BUILD_DIR)
 deployments: $(BUILD_DIR)/kustomize $(BUILD_DIR)/kubernetes-split-yaml manifests generate ## Generate the deployment files with kustomize
 	$(BUILD_DIR)/kustomize build --reorder=none deploy/overlays/cluster -o deploy/operator.yaml
 	$(BUILD_DIR)/kustomize build --reorder=none deploy/overlays/namespaced -o deploy/namespace-operator.yaml
+	cp -f deploy/base/certificate.yaml deploy/overlays/openshift/no_certificate.yaml
```

Comment:
@JAORMX this feels like a bit of a hack, but I wanted to make sure we include nothing from the certificate.yaml file. Just pointing this out myself for review..

Reply:
I'm not sure I get the reasoning for this. the overlay should have everything to build the `openshift.yaml` and `openshift-dev.yaml` files.